### PR TITLE
Clean up xrefs

### PR DESCRIPTION
**Pull request by cthoyt**
This PR cleans up xrefs in a few ways:

1. Remove some spurious xrefs that look like they were imported (e.g., ID:KFC0000604, JB:def)
2. Fixes some typos
3. Removes garbage entries (e.g., TODO:TODO, TEMP:MP)

### CODE REVIEW

**Review by shawntanzk — APPROVED** [review]

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -120472,7 +120472,7 @@ is_a: UBERON:0000468 ! multicellular organism
 [Term]
 id: UBERON:0007198
 name: hermaphrodite anatomical structure
-def: "An anatomical structure present only in hermaphrodites." [JB:DEF]
+def: "An anatomical structure present only in hermaphrodites." []
```
[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:120475]: Oh yeah, I dont know when this got triggered, but defs need an xref now - I remember fixing a bunch of this. Happy to try to find out who JB is, if not previously I used ID range to figure it out. Will check

[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:120475]: This falls within @cmungall range - do you know who JB is, if not could we use your ORCID? thanks

[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:120475]: we should really do a proper overhaul of orcids to https - I followed the one before and it was HTTP >.<

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
@shawntanzk this can be merged at your discretion

**Pull request merged**